### PR TITLE
Visual regression blog

### PR DESCRIPTION
**Pull request by wswebcreation**
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Blog post

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

 @webdriverio/technical-committee

### CODE REVIEW

File: website/blog/2019-05-18-visual-regression-for-v5.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+            blockOutStatusBar: true,
+            blockOutToolBar: true,
+            // ... more options
+        }], 
+    ],
+    // ...
+};
+``` 
+
+More plugin options can be found [here](./docs/OPTIONS.md#plugin-options).
```
Comment (christian-bromann) [inline website/blog/2019-05-18-visual-regression-for-v5.md]:
```suggestion
More plugin options can be found [here](/docs/options.html).
```

Also I don't see a plugin-option section

Comment (christian-bromann) [reply website/blog/2019-05-18-visual-regression-for-v5.md]:
```suggestion
More plugin options can be found [here](https://github.com/wswebcreation/wdio-image-comparison-service/blob/master/docs/OPTIONS.md#plugin-options).
```

Ah, I think you want to set the full link url

**Comment by wswebcreation** [comment]
@christian-bromann

Tnx for the changes, I've committed them. 
If agreed this can be merged.

**Comment by anselmbradford** [comment]
Why is this not included in the Services in the left-hand nav of https://webdriver.io/docs/gettingstarted.html ?

**Comment by christian-bromann** [comment]
@anselmbradford would you mind making a PR to add this to our docs?

**Comment by anselmbradford** [comment]
@christian-bromann so to confirm, https://github.com/wswebcreation/wdio-image-comparison-service, is the official webdriver visual regression testing solution?

Also, how are services added to the sidebar on the website? I see https://github.com/webdriverio/webdriverio/blob/13235196d34f62456103d00a27b45f6fb30e6909/website/_sidebars.json but it doesn't include the services section.

**Comment by christian-bromann** [comment]
"official" is probably a hard word, it is one of the services that currently works very well with the latest WebdriverIO project.

Would you mind adding it to the sidebar by following this PR: https://github.com/webdriverio/webdriverio/pull/4976/files ?

**Pull request merged**